flink-iotdb-connector/src/main/java/org/apache/iotdb/flink/Event.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.iotdb.flink;
+
+public class Event {
+    private String device;
+    private Long timestamp;
+    private String measurement;
```

Comment:
In many applications, when a monitored device sends data to the server, it does not just send one data point. 
Instead, it sends a "row", i.e., several measurements and their values. (And in some applications, they even sends data with several timestamps, but it is not very common.) 
As far as I know, a device may have more than 100 measurements in the industrial applications. In this case, splitting a "row" into many Events may burden Flink as well as IoTDB (because there are too many small TCP packets).

How do you think?

Reply:
Make sense. We should support most scenarios. Will update. 
@jixuan1989 Thanks for your input.